samples/ai-meeting-helper/src/index.js

```diff
@@ -0,0 +1,267 @@
+}
+
+server.post("/EventHandler", async (req, res) => {
+  if(req.url === "/EventHandler")
+  {
+    var notification = req.body.value;
+    var responsePayload = await DecryptionHelper.processEncryptedNotification(notification);
+    var requestId  = "";
+    if(responsePayload.eventType === "Microsoft.Communication.CallEnded")
+      {
+        await serverAPI.getMeetingTranscription(req.body.value[0].resource, req.body.value[0].clientState)
+        .then(async meetingDetails => {
+           
+            requestId = req.rawHeaders[3];
+           
+            if (alreadyProcessed(requestId)) {
+              console.log(`Request ${requestId} already processed`);
+               return res.status(200).end();
+            }
+            
+            // Process the webhook payload
+            console.log('Processing webhook payload:', req.body);
+          
+            // Call graph API        
+            serverAPI.getMeetingDetailsUsingSubscription(req.body.value[0].resource, req.body.value[0].clientState)
+            .then(async onlineMeetingDetail => {
+                 // get User and meeting details
+              const UserInfo =  GetUserInformationAndSendActionItems(req.body.value[0].resource,  meetingDetails, onlineMeetingDetail, req.body.value[0].subscriptionId);
+            
+            });
+    
+            // Mark this request ID as processed
+            markAsProcessed(requestId);
+            
+            return res.status(200).end();
+        });
+  }
+}
+ else if (req.url != undefined && req.url.split('=').length>-1) {
+    let url = decodeURIComponent(req.url.split('=')[1]);
+    url = url.replace(/\+/g, ' ');
+    res.send(url);
+  }
+  else {
+      eventUpdated = true;
+      res.status(200).send('OK');
+  }
+});
+
+server.post("/api/messages", async (req, res) => { 
+
+  // const c = await serverAPI.transcribeAndExtractUserActionItems("","", "");
+  // return "";
+
+    // Route received a request to adapter for processing
+    await adapter.process(req, res, async (context) => {
+      if (context.activity.type === 'message') { 
+        if(context.activity.text.indexOf('upcoming')>-1)
+          { 
+            const meetings = await serverAPI.getTodaysMeetingAgenda(req,context);
+            // Your message handling logic goes here
+            await context.sendActivity({ attachments: [meetings] });
+          }       
+      }
+      else if (context.activity.type === 'invoke') {  
+          
+          let subscription = "";
+          try
+          {
+            await context.sendActivity({
+              type: 'invokeResponse',
+              value: {
+                  status: 200,
+                  body: {
+                      message: "action proceeded"
+                  }
+              }
+            });   
+
+            subscription = await serverAPI.createSubscription(context.activity.value.id, context.activity.from.aadObjectId , context.activity.conversation.id);
```

Comment:
It can be subscriptionAPI rather than serverAPI

Reply:
Didn't get this point, Should be need to create separate class for this?